js/message.js

```diff
@@ -0,0 +1,54 @@
+/*Сообщения при отправке данных*/
+
+import {isEscapeKey} from './util.js';
+
+const errorTemplate = document.querySelector('#error').content.querySelector('.error');
+const successTemplate = document.querySelector('#success').content.querySelector('.success');
+
+const messageType = () => document.querySelector('.error, .success');
+
```

Comment:
учитывая что это функция нужен глагол, 
```suggestion
const getMessageType = () => document.querySelector('.error, .success');

```